packages/rollup-plugin/src/index.ts

```diff
@@ -81,14 +85,18 @@ export function vanillaExtractPlugin({
 
       // ...replace import paths with relative paths to emitted css files
       const chunkPath = dirname(chunkInfo.fileName);
-      return importsToReplace.reduce((codeResult, importPath) => {
+      const output = importsToReplace.reduce((codeResult, importPath) => {
         const assetId = emittedFiles.get(importPath)!;
         const assetName = this.getFileName(assetId);
         const fixedImportPath = `./${normalize(
           relative(chunkPath, assetName),
         )}`;
         return codeResult.replace(importPath, fixedImportPath);
       }, code);
+      return {
+        code: output,
+        map: chunkInfo.map ?? null,
```

Comment:
Is `null` handled differently than `undefined`?

Reply:
Yes it seems so. When I tested it `chunkInfo.map` actually was undefined and still resulted in the same warning. But with `null` it is suppressed.